src/util/resource_manager.h

```diff
@@ -200,8 +197,8 @@ class CVC4_PUBLIC ResourceManager
   /** Counter indicating how often to check resource manager in loops */
   static const uint64_t s_resourceCount;
 
-  /** Receives a notification on reaching a limit. */
-  ListenerCollection d_listeners;
+  /** Receives a notification on reaching a resource out. */
```

Comment:
It may still be a (per-call) timeout. The only thing that was removed was the cumulative time limit.